src/pages/candidate-details/candidate-details.selector.ts

```diff
@@ -22,6 +22,12 @@ export const isCandidateCommentsEmpty = (slot: any): boolean => {
   return isEmpty(slot.booking.application.specialNeeds) && isEmpty(slot.booking.previousCancellation);
 };
 
+export const getCandidateId = (slot: any): string => `${slot.booking.candidate.candidateId}`;
```

Comment:
Just out of curiosity, why is this wrapped in a template string?

Reply:
Hmmm. No reason for it at all that I can think of. Will remove now.